rsocket-core/src/main/java/io/rsocket/RSocketServer.java

```diff
@@ -168,6 +179,7 @@ private synchronized void cleanUpChannelProcessors() {
           return metadataPush(new PayloadImpl(frame));
         case LEASE:
           // Lease must not be received here as this is the server end of the socket which sends leases.
```

Comment:
Guess this comment is not valid as Connection.receive() is constrained as Unicast so has to be handled here anyways?